src/features/l2ps-messaging/L2PS_MESSAGING_QUICKSTART.md

```
@@ -0,0 +1,426 @@
+- **Conversation history** — Queryable from the node with authenticated proof
+
+---
+
+## 1. Prerequisites
+
+### L2PS Network
+
+An L2PS network must be set up and running. See [L2PS Quick Start](../../libs/l2ps/L2PS_QUICKSTART.md) for details.
+
+Verify your L2PS network is loaded:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)